next-docs/pages/world/systems.mdx

```diff
@@ -1 +1,111 @@
 # Systems
+
+One of the design principles of MUD is to separate the state of the `World` from the business logic that it implements.
+We do this by having stateless `System` contracts with the functions that implement the business logic.
+These contracts are called through the `World`, and call back on the `World` when they need to read or modify state information, which is stored in [tables](./tables).
+
+<details>
+
+<summary>Detailed illustration</summary>
+
+![Detailed illustration](./world-table.svg)
+
+1. An external entity (contract or externally owned account) calls on the `World` a function called `myns_sys_fun`.
+   This means the `fun` function in the `sys` system, which is part of the `myns` namespace (the system is also refered to as `myns:sys`).
+
+1. The `World` verifies that access is permitted (for example, because `myns:sys` is publicly accessible) and if so calls `fun` on the `myns:sys` contract with the provided parameters.
+
+1. At some point in its execution `fun` decides to update the data in the table `myns:tab`.
+   As with all other tables, this table is stored in the `World`'s storage.
+   To modify it, `fun` calls a function on the `World` contract.
+
+1. The `World` verifies that access is permitted (by default it would be, because `myns:sys` has access to the `myns` namespace).
```

Comment:
```suggestion
1. The `World` verifies that access is permitted (in this example it would be, because [systems have access to their namespace](./namespace-access-control)).
```